be/src/common/status.h

```diff
@@ -157,6 +157,7 @@ class Status {
     bool is_mem_limit_exceeded() const { return code() == TStatusCode::MEM_LIMIT_EXCEEDED; }
     bool is_thrift_rpc_error() const { return code() == TStatusCode::THRIFT_RPC_ERROR; }
     bool is_end_of_file() const { return code() == TStatusCode::END_OF_FILE; }
+    bool ok_or_eof() const { return ok() || is_end_of_file(); }
```

Comment:
better to keep the sematics of is_xxx function simple and do not introduce this combination of judgement